### PR TITLE
Add `threshold` and `signal` details

### PR DESCRIPTION
**Pull request by tomayac**

### CODE REVIEW

File: mdn-drafts/QUICK-REFERENCE.md [hunk 1]
```diff
@@ -54,7 +54,9 @@ acceptance and `"denied"` on refusal.
 
 Returns a `Promise` that resolves when the detector has started listening for
 changes in the user's idle state. `userState` and `screenState` are populated
-with their initial values.
+with their initial values. Takes an optioanl options object with the
```
Comment (jpmedley) [inline mdn-drafts/QUICK-REFERENCE.md:57]:
```suggestion
with their initial values. Takes an optional options object with the
```

Comment (jpmedley) [reply mdn-drafts/QUICK-REFERENCE.md:57]: I can't merge this to your PR.

**Review by jpmedley — CHANGES_REQUESTED** [review]
The expanded examples are worth a lot. Thanks

**Review by reillyeon — APPROVED** [review]

**Comment by tomayac** [comment]
Happy to help. This looks good to merge then.

**Pull request merged**